cni-plugin/Makefile

```diff
@@ -96,7 +96,7 @@ else
 endif
 
 $(BIN)/calico-ipam: $(BIN)/calico
-	cp "$<" "$@"
+	ln -sf "./calico" "$@"
```

Comment:
Should this be `$(BIN)/calico`?

Reply:
No, we should use the relative path of calico to make sure it works after copy to cni image.

Reply:
this doesn't works
```bash
❯ pwd
/Volumes/Projects/go/src/github.com/projectcalico/calico
❯ cd cni-plugin
❯ ln -sf bin/amd64/calico bin/amd64/calico-ipam
❯ readlink -f bin/amd64/calico-ipam
```

this works
```
❯ ln -sf ./calico bin/amd64/calico-ipam
❯ readlink -f bin/amd64/calico-ipam
/Volumes/Projects/go/src/github.com/projectcalico/calico/cni-plugin/bin/amd64/calico
```

Reply:
oh yes, of course. Sorry for the noise.